libraries/AP_Scripting/examples/mission-load.lua

```diff
@@ -0,0 +1,65 @@
+-- Example of loading a mission from the SD card using Scripting
+-- Would be trivial to select a mission based on scripting params or RC switch
+
+local function read_mission(file_name)
+
+  -- Open file
+  local file = io.open(file_name)
```

Comment:
I think we need to determine the results of the process.
I think it's better to specify the read attribute.

Reply:
It defaults to read only